### DISCUSSION TITLE
Custom Data Point for LineChart in the BarChart

### DISCUSSION THREAD

**Discussion by ThucTrieu** [Abhinandan-Kushwaha/react-native-gifted-charts#346] [Q&A] [answered]
Can we custom data point for line chart in the bar chart?
I tried: 
<img width="463" alt="image" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/100109060/9358945c-2a33-4301-b17b-4caf20a0fbe4">
<img width="356" alt="image" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/100109060/5470d421-2e8b-41e1-98b0-569faba95835">
but don't work

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @ThucTrieu 👋
Thanks for pointing this out. Custom data points are currently not supported for LineChart in BarChart. I will add this feature in next release which will probably be out within a week.

**Reply by ThucTrieu** [reply]
Thanks @Abhinandan-Kushwaha , If you don't mind, please support it as soon as possible, waiting for information from you
